apps/src/code-studio/components/progress/UnitOverviewHeader.jsx

```diff
@@ -20,8 +20,28 @@ import {assignmentCourseVersionShape} from '@cdo/apps/templates/teacherDashboard
 import ParticipantFeedbackNotification from '@cdo/apps/templates/feedback/ParticipantFeedbackNotification';
 import VerifiedResourcesNotification from '@cdo/apps/templates/courseOverview/VerifiedResourcesNotification';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {pegasus, studio} from '../../../lib/util/urlHelpers';
 
 const SCRIPT_OVERVIEW_WIDTH = 1100;
+// Scripts that users will be warned are outdated and have been succeeded by others.
+// This object in the form of {scriptName : description of warning}.
+const OUTDATED_SCRIPT_NAMES_AND_DESC = {
+  course1: i18n.outdatedCourseWarningDescCourses1To4({
+    csFundCourseLink: pegasus('/educate/curriculum/csf-transition-guide')
```

Comment:
NIT: I was reading this during the fundraising conversation at all-staff and therefore thought it was "cs funding course link" rather than "cs fundamentals course link." Something like "csfCourseLink" or "csFundamentalsCourseLink" would be more clear.

Reply:
Ah. I see this is an already-existing variable name, so nevermind.